kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
Sure: https://github.com/istio/manager/blob/master/proxy/envoy/policy.go#L63
The spreadsheet format needs to be converted to a proto or something that can be programmatically reused.